cosmo_tester/test_suites/ha/ha_cluster_of_three_test.py

```diff
@@ -0,0 +1,207 @@
+########
+# Copyright (c) 2017 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+import pytest
+from cosmo_tester.framework.examples.hello_world import HelloWorldExample
+from cosmo_tester.framework.cluster import CloudifyCluster
+from .ha_helper import HighAvailabilityHelper as ha_helper
+
+
+@pytest.fixture(scope='function')
+def cluster(
+        request, cfy, ssh_key, module_tmpdir, attributes, logger):
+    """Creates a HA cluster from an image in rackspace OpenStack."""
+    logger.info('Creating HA cluster of 2 managers')
+    cluster = CloudifyCluster.create_image_based(
+        cfy,
+        ssh_key,
+        module_tmpdir,
+        attributes,
+        logger,
+        number_of_managers=3,
+        create=False)
+
+    # manager2 - Cloudify latest - don't install plugins
+    cluster.managers_config[1].upload_plugins = False
+
+    # manager3 - Cloudify latest - don't install plugins
+    cluster.managers_config[2].upload_plugins = False
+
+    cluster.create()
+
+    try:
+        encryption_key = ha_helper.create_encription_key()
+        manager1 = cluster.managers[0]
+        manager2 = cluster.managers[1]
+        manager3 = cluster.managers[2]
+
+        manager1.use()
+        cfy.cluster.start(timeout=600,
+                          cluster_host_ip=manager1.ip_address,
+                          cluster_node_name='m1',
```

Comment:
please note `ha_helper.verify_nodes_status` expects the node name to be manager.index so this will currently break